<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptOfSearchDialog</class>
 <widget class="QDialog" name="OptOfSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>364</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Properties Of Directory</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="protect">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>341</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Information</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Have Index file Count:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Waiting Index file Count:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="indexedfileCount">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="waitIndexfileCount">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="updateBtn">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>261</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Update Index file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reCreateBtn">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>261</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ReCreate Index file</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>391</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="temp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>41</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
